glossary/en/terms/re-use/index.md

```diff
@@ -5,4 +5,4 @@ title: Re-use
 ---
 
 
-It is rare that data gathered for a particular purpose does not have other possible uses. Happily, data is an infinite resource (see [tragedy of the anti-commons](/glossary/en/terms/tragedy-of-the-anti-commons/)); once gathered, for whatever reason, it can be re-used again and again, in ways that were never envisaged when it was collected, provided only that the data-holder makes it available under an open licence to enable such re-use.
```

Comment:
@morchickit I think it's not a bug. Actually, the second `)` is matched with the opening parenthesis before `see [tragedy ...`